changelog.txt

```diff
@@ -39,7 +40,7 @@ Added
 - Add support for aggregation (operations acting on multiple jobs) via ``flow.aggregator`` (#464, #516, #542).
 - Add official support for Andes cluster (#500).
 - Decorator for setting directives while registering operation function ``FlowProject.operation.with_directives`` (#309, #502).
-- Feature to install execution hooks for the automated execution of functions with operations (#28, #189, #508).
+- Add new flow command ``flow template create`` for automatic creation of custom templates (#520, #534).
```

Comment:
This looks like it's not properly rebased.

Reply:
Shouldn't this,

> - Feature to install execution hooks for the automated execution of functions with operations (#28, #189, #508).

be in the changelog entry for this branch be under the unreleased version 0.16.0 instead of the released version 0.15.0?

Reply:
Yes. This PR’s development time has spanned multiple releases, so the changelog messages need to be moved to an unreleased version.